packages/xrpl/src/sugar/submit.ts

```diff
@@ -52,12 +52,70 @@ async function submit(
  * validated ledger (or has errored/will not be included for some reason).
  * See [Reliable Transaction Submission](https://xrpl.org/reliable-transaction-submission.html).
  *
+ * @example
+ *
+ * ```ts
+ * const { Client, Wallet } = require('xrpl')
+ * const client = new Client('wss://s.altnet.rippletest.net:51233')
+ *
+ * async function submitTransaction() {
+ *   const senderWallet = client.fundWallet()
+ *   const recipientWallet = client.fundWallet()
+ *
+ *   const transaction = {
+ *     TransactionType: 'Payment',
+ *     Account: senderWallet.address,
+ *     Destination: recipientWallet.address,
+ *     Amount: '10'
+ *   }
+ *
+ *   try {
+ *     await client.submitAndWait(signedTransaction, { wallet: senderWallet })
+ *     console.log(result)
+ *   } catch (error) {
+ *     console.error(`Failed to submit transaction: ${error}`)
+ *   }
+ * }
+ *
+ * submitTransaction()
+ * ```
+ *
+ * In this example, we create two new test accounts, one for the sender and one for the recipient.
+ * We then use them as the account sending the payment, and the recipient of the funds.
+ *
+ * Next we create a transaction object that represents a payment of 10 XRP from the sender account to the
+ * recipient account. We set the Amount property to 10 XRP and leave the other properties empty.
+ *
+ * The transaction object is passed to the `autofill()` method of the `Client` class, which fills in the
+ * missing fields based on the current state of the XRP Ledger.
+ *
+ * Continuing on we then pass the filled-in transaction object to the `sign()` method of the `Wallet` class
+ * using `await` to wait for the method to complete. If the signing is successful, the resulting signed
+ * transaction blob is passed to the `submitAndWait()` method of the `Client` class.
+ *
+ * The `submitAndWait()` method submits the signed transaction blob to the XRP Ledger and waits for the transaction
+ * to be included in a validated ledger before resolving or rejecting the returned promise. If the transaction is
+ * successfully validated and included in a ledger, the promise resolves with the transaction result object. If there's
+ * an issue submitting or validating the transaction, the promise rejects with an error.
+ *
+ * If the promise resolves, the function logs a success message to the console that includes the transaction hash. If the
+ * promise rejects, the function logs an error message to the console using console.error().
+ *
```

Comment:
```suggestion
 * In this example we submit a payment transaction between two newly created testnet accounts. 
 * 
 * Under the hood, `submit` will call `client.autofill` by default, and because we've passed in a `Wallet` it
 * Will also sign the transaction for us before submitting the signed transaction binary blob to the ledger. 
 * 
 * This is similar to `submitAndWait` which does all of the above, but also waits to see if the transaction has been validated.
```